lib/expressView.js

```diff
@@ -30,9 +30,13 @@ function ReactEngineView(name, options) {
   // when the view name starts with `/` we assume
   // that we need to use react router to render.
   this.useRouter = (name[0] === '/');
-
+  this.addJsxExt = false;
   if (this.useRouter) {
     name = url.parse(name).path;
+    if(name.indexOf('.') !== -1){
+      name += '.jsx';  
```

Comment:
@bodyscripter can you please explain what is being done here? we cannot assume that all files will have `jsx` extension